src/main/java/org/opensearch/security/rest/DashboardsInfoAction.java

```diff
@@ -83,9 +71,13 @@ public DashboardsInfoAction(final Settings settings, final RestController contro
         this.evaluator = evaluator;
     }
 
-    @Override
     public List<Route> routes() {
-        return routes;
+        return ImmutableList.of();
```

Comment:
same as above. Why keep it when you removed from other files

Reply:
My bad, they should be deleted as well, fixed